IHP/IDE/SchemaDesigner/View/Layout.hs

```diff
@@ -287,17 +287,67 @@ renderObjectSelector statements activeObjectName = [hsx|
         |]
             where
                 contextMenuId = "context-menu-" <> tshow id
-        renderObject Comment {} id = mempty
```

Comment:
Why have these empty statements been removed? The idea was that we hide all the statements that the user cannot interact with from the GUI

Reply:
My train of thought was: icons look nice -> oh, we only show tables and enums -> let's show more (all) to see more pretty icons

Reply:
I agree, but in big project's this really wastes a huge amount of screen space (I've tested it)

Reply:
We could sort the entries by type (e.g. tables, types, indexes), add a sub-header and by default collapse everything except tables. That's also how pgadmin and dbeaver work.

(FWIW, I'm also fine with simply hiding everything except tables, types and indexes)

Reply:
This is all too complicated for things that cannot be used from the Schema Designer. Let's keep the behaviour from master and hide everything which cannot be interacted with